mirrord/config/src/env.rs

```diff
@@ -58,10 +57,10 @@ use crate::{
 /// [feature.env]
 /// exclude = "USER;SECRET"
 /// ```
-#[derive(MirrordConfig, Default, Deserialize, PartialEq, Eq, Clone, Debug, JsonSchema)]
-#[serde(deny_unknown_fields)]
-#[config(map_to = EnvConfig)]
-pub struct EnvFileConfig {
+#[derive(MirrordConfig, Clone, Debug)]
+#[config(map_to = "EnvFileConfig", derive = "JsonSchema")]
```

Comment:
This one is more of a question rather than an actual suggestion:

Do you prefer having these arguments be `Lit` and not `Ident`? I'm ok with either (slight bias towards `Ident`, but again, it's ok as-is).

Reply:
I wanted them to be Ident but it doesn't work, If I don't specify it as a Lit it simply doesn't appear in result of `parse_meta`

Reply:
Yeah, that's probably why many projects tend to go with `Lit`. Again this is fine, will mark as resolved.